### PR TITLE
added docker

### PR DESCRIPTION
**Pull request by c-kalenyk**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      - $SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT
+      - $DEBUG_PORT:$DEBUG_PORT
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url"  : "jdbc:mysql://mysqldb:$MYSQL_DOCKER_PORT/$MYSQL_DATABASE?serverTimezone=UTC",
+        "spring.datasource.username" : "$MYSQL_USER",
+        "spring.datasource.password" : "$MYSQL_ROOT_PASSWORD",
+        "spring.jpa.hibernate.ddl-auto" : "validate"
+      }'
+      JAVA_TOOL_OPTIONS: "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (okuzan) [inline docker-compose.yml]: 5005 is DEBUG PORT

**Review by okuzan — CHANGES_REQUESTED** [review]
git rm --cached .env
add it to gitignore, don't expose it
create .env.template for env var names, without values

File: .gitignore [hunk 2]
```diff
@@ -3,6 +3,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
+git rm --cached .env
```
Comment (sarakhmen) [inline .gitignore]: ? :)

File: .env
Comment (sarakhmen) [inline .env:1]: don't expose this file as Oleh advised

**Review by sarakhmen — CHANGES_REQUESTED** [review]

**Review by Elena-Bruyako — APPROVED** [review]
well done

**Pull request merged**